### PR TITLE
Link Style 

### PR DESCRIPTION
**Pull request by Fatmasiam**
#19

### CODE REVIEW

File: Client/src/components/common/Link/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import './index.css'
 
 export default function link({ title, to }) {
```
Comment (AsalaKM) [inline Client/src/components/common/Link/index.js]: The name of your function should be capital letter
`export default function Link`

Comment (Fatmasiam) [reply Client/src/components/common/Link/index.js]: done

Comment (Fatmasiam) [reply Client/src/components/common/Link/index.js]: done

**Review by AbdallahAmmar96 — APPROVED** [review]

File: Client/src/components/common/Link/index.js [hunk 2]
```diff
@@ -3,9 +3,9 @@ import { Link } from 'react-router-dom';
 
 import './index.css'
 
-export default function link({ title, to }) {
+export default function SubLink({ title, to }) {
   return (to ?
```
Comment (kannanhassouna) [inline Client/src/components/common/Link/index.js:7]: awesome. 
good job in taking care of pitfalls

**Review by kannanhassouna — APPROVED** [review]

**Review by AsalaKM — APPROVED** [review]

File: Client/src/components/common/Link/index.js [hunk 3]
```diff
@@ -3,9 +3,9 @@ import { Link } from 'react-router-dom';
...
-    <Link to={to}>
+    <Link  className='sidebar-linklist-link'to={to}>
```
Comment (kannanhassouna) [inline Client/src/components/common/Link/index.js:8]:
```suggestion
    <Link  className='sidebar-linklist-link' to={to}>
```
for readability reasons

**Review by kannanhassouna — APPROVED** [review]

**Review by naremanhilles — APPROVED** [review]

**Pull request merged**